queue_processors/autoreduction_processor/reduction_exceptions.py

```diff
@@ -0,0 +1,28 @@
+# ############################################################################### #
+# Autoreduction Repository : https://github.com/ISISScientificComputing/autoreduce
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+# ############################################################################### #
+"""
+Exceptions that may be raised when performing a reduction
+"""
+
+
+class DatafileError(Exception):
+    """
+    Exception when there is an issue reading a datafile
+    """
+
+
+class ReductionScriptError(Exception):
+    """
+    Exception raised when an unhandled exception is produced within the reduction script
+    """
+
+
+class SkippedRunException(Exception):
```

Comment:
I like created a separate files for exceptions. However, is SkippedRunException and exception any more? I.e. now that we report skipped runs in webapp has skipped run become part of normal operation